CONTRIBUTING.rst

```diff
@@ -76,7 +76,7 @@ Setup
 ~~~~~
 
 -  Clone and register the package for development as described in the
-   `README <README.md#installation>`__
+   `here <http://qcodes.github.io/Qcodes/start/index.html#installation>`__
```

Comment:
here does not seem to match the line above.

Reply:
I removed "in the" and keep "here" for the link to be coherent with the style of README.rst.